### DISCUSSION TITLE
This Key is not available, To Use it, you must import it as one of your own!

### DISCUSSION THREAD

**Discussion by David-WDavid-W** [zeapo/Android-Password-Store#1513] [Q&A]
This question was asked a few years ago and the discussion was closed without a solution. Two people added their questions since then, but no answer.
So... On Android I am using FairEmail and OpenKeyChain. I have imported keys, but they do not show up in "My Keys", and I cannot locate a menu item to move there. When I try decrypting a message in FairEmail with any of the keys I have imported, I get the message above.
Thanks

**Comment by msfjarvis** [comment]
This is not an OpenKeychain support forum, please go to https://github.com/open-keychain/open-keychain for queries not related to Android Password Store.
